src/Raven.Studio/typescript/components/pages/database/settings/documentSchema/partials/ValidationSchemaViewSheetPanel.tsx

```diff
@@ -609,3 +657,19 @@ function ValidationCollectionAccordionItem({
         </Accordion.Item>
     );
 }
+
+function getIsCollectionSelected(name: string, selectedCollections: Record<string, boolean>): boolean {
+    const explicit = selectedCollections[name];
+    if (typeof explicit === "boolean") {
+        return explicit;
+    }
+
+    return true;
```

Comment:
`return selectedCollections[name] === true`

Reply:
This code is correct. If explicit is undefined, function return `true` (checkbox is marked).

Reply:
I can simplify the code in this way:

```suggestion
    if (selectedCollections[name] != null) {
        return selectedCollections[name];
    }

    return true;
```

Reply:
Wouldn't it be easier to set the initial state to all selected?